modules/cf-core/src/machine/index.ts

```diff
@@ -15,35 +15,35 @@ import {
   UninstallParams,
   UninstallVirtualAppParams,
   UpdateParams,
-  WithdrawParams
+  WithdrawParams,
```

Comment:
This should use CF's lint rules. Our lint rules are conflicting with theirs, and we want to be able to PR this stuff upstream and absorb upstream changes. I added their lint config to the modules in this commit:

https://github.com/ConnextProject/indra/pull/568/commits/b0b4ce99d1462e392792f9aff8fa8325b63071a4

It's stuck in my branch I realized. I'll cherry-pick this into staging.

Reply:
^^ please 💯